### DISCUSSION TITLE
Python interface of LAMMPS is not compiled in the 2.0.3 version？

### DISCUSSION THREAD

**Discussion by Epikurz** [deepmodeling/deepmd-kit#1456] [General]
There is pre-built Python interface of LAMMPS in the Deepmd-kit 2.0 installed by offline package. In the latest version, the interface is not available. 

Is it possible to recover the feature in next version?

**Comment by njzjz** [comment]
Fixed by https://github.com/deepmd-kit-recipes/lammps-dp-feedstock/pull/48.

One can manually update via `conda update lammps-dp -c https://conda.deepmodeling.org`.
